test/acceptance/editors-integration-test.js

```diff
@@ -31,16 +35,17 @@ describe('editors integration', function () {
   });
 
   describe('reading from stdin', function () {
-    it('prints valid JSON strings with error', function () {
+    it('prints valid JSON strings with error', async function () {
       setForBasicDebuggerError(project);
 
-      let result = run(
-        project,
-        ['--json', '--filename', 'template.hbs', '/dev/stdin', '<', 'template.hbs'],
-        {
-          shell: true,
-        }
-      );
+      const args =
+        process.platform === 'win32'
+          ? ['--json', '--filename', 'template.hbs', '<', 'template.hbs']
+          : ['--json', '--filename', 'template.hbs', '/dev/stdin', '<', 'template.hbs'];
```

Comment:
It is fine to remove `/dev/stdin` at all from these args. It will allow to remove this ternary.

`ember-template-lint --json --filename template.hbs < template.hbs` is the same command than `ember-template-lint --json --filename template.hbs /dev/stdin < template.hbs` or `ember-template-lint --json --filename template.hbs - < template.hbs`.

Using /dev/stdin in a command is already tested in `cli-test.js`.